README.md

```diff
@@ -28,19 +28,24 @@ import ytch from 'yt-channel-info'
 
 ## API
 
-**getChannelInfo(channelId, [channelIdType])**
-
-Returns information about a given channel ID.
-The optional argument 'channelIdType' can be provided to get faster results and less network requests if the type of channel id is known.
-- `0` = Default value used by the module. It will try all url types in the order channel -> user -> name
-- `1` = A channel id that is used with `https://www.youtube.com/channel/channelId` urls
-- `2` = A user id that is used with `https://www.youtube.com/user/channelId` urls
-- `3` = A name id that is used with `https://www.youtube.com/c/channelId` urls
+**getChannelInfo(payload)**
```

Comment:
I am fine updating all API methods to use payload
But we need to remember to update the code on FT when we update this package

Reply:
Actually should we have a notice of this breaking change if merged?

Reply:
> a notice of this breaking change

I don't get what you mean.

Reply:
nvm :facepalm: